packages/dispatcher-converter/src/converter/amsDispatcherConfigConverter.js

```diff
@@ -1789,6 +1813,66 @@ class AEMDispatcherConfigConverter {
 
         return usedFiles;
     }
+    /**
+     * Scans the `available_vhosts` folder, and if a symbolic link doesn't already exist in the `enabled_vhosts` folder, one
+     * is created.
+     */
+    createVhostSymLinks(enabledVhostsPath, availableVhostsPath) {
+        let conversionStep = this.createVhostSymLinksSummaryGenerator();
+        let files = glob.sync(path.join(availableVhostsPath, "*.vhost")) || [];
+        files.forEach((file) => {
+            logger.info(
+                "Creating Vhost Symbolic Link in target folder for file : " +
+                    file
+            );
+            let fileName = path.basename(file);
+            this.FileOperationsUtility.createSymLink(
+                "../available_vhosts/" + fileName,
+                path.join(enabledVhostsPath, fileName),
+                fileName,
+                conversionStep
+            );
+        });
+        this.conversionSteps.push(conversionStep);
+    }
+
+    createVhostSymLinksSummaryGenerator() {
+        logger.info("Creating Vhost SymLinks");
+        return new ConversionStep(
+            "Create Vhost SymLinks",
+            "Generate Vhost SymLinks form vhost files"
+        );
+    }
+
+    /**
+     * Scans the `available_farms` folder, and if a symbolic link doesn't already exist in the `enabled_farms` folder, one
+     * is created.
+     */
+    createFarmSymLinks(enabledFarmsPath, availableFarmsPath) {
+        let conversionStep = this.createFarmSymLinksSummaryGenerator();
+        let files = glob.sync(path.join(availableFarmsPath, "*.farm")) || [];
+        files.forEach((file) => {
+            logger.info(
+                "Creating Farm Symbolic Link in target folder for file : " +
+                    file
+            );
+            this.FileOperationsUtility.createSymLink(
+                "../available_farms/" + path.basename(file),
+                path.join(enabledFarmsPath, path.basename(file)),
+                path.basename(file),
```

Comment:
Do we really need to pass the filename (it can easily be computed in the `createSymLink` method itself)? Would help reduce the number of parameters in the method signature.